tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -191,17 +192,23 @@ def isStringLargerThan(val):
         lambda value: f'{value} is not larger than {val}.'
     )
 
+
+def _is_empty(value, start, end):
+    return value_is_empty(value[start:end], end - start)
+
+
 def notEmpty(start=0, end=None):
     """Validate that string value isn't only blanks."""
+    # asdf = not str(value)[start:end if end else len(str(value))].isspace()
     return make_validator(
-        lambda value: not str(value)[start:end if end else len(str(value))].isspace(),
+        lambda value: not _is_empty(str(value), start, end if end else len(value)),
```

Comment:
and update the `notEmpty` and `isEmpty` (below) validators to use the shared implementation